examples/kbot/walking.py

```diff
@@ -299,6 +297,35 @@ def sample_and_log_prob(
         return action, log_prob
 
 
+class KBotZeroActions(ActionModel):
+    mlp: MLP
+
+    def setup(self) -> None:
+        self.log_std = self.param("log_std", nn.initializers.constant(-0.7), (NUM_OUTPUTS,))
+
+    def __call__(self, obs: FrozenDict[str, Array], cmd: FrozenDict[str, Array]) -> Array:
+        x_n = jnp.concatenate([obs_array for obs_array in obs.values()], axis=-1)
+        cmd_n = jnp.concatenate([cmd_array for cmd_array in cmd.values()], axis=-1)
+        x_n = jnp.concatenate([x_n, cmd_n], axis=-1)
+        actions_n = self.mlp(x_n)
+
+        return actions_n
+
+    def calc_log_prob(self, prediction: Array, action: Array) -> Array:
+        mean = prediction
+        std = jnp.exp(self.log_std)
+
+        log_prob = -0.5 * jnp.square((action - mean) / std) - jnp.log(std) - 0.5 * jnp.log(2 * jnp.pi)
+        return jnp.sum(log_prob, axis=-1)
```

Comment:
btw @WT-MM @michael-lutz @alik-git - i am a big fan of using noam shazeer's shape suffixes convention for variable naming. even though it results in strange-looking variable names, i believe it greatly improves readability

blog post: https://medium.com/@NoamShazeer/shape-suffixes-good-coding-style-f836e72e24fd

so for example, you would want to write something like `predictions_n` and `actions_n` to indicate that they are 1D tensors with the last dimension being the dimensionality of the gaussian